functions_task_arithmetic_function.py

```diff
@@ -0,0 +1,48 @@
+# 1. Write an arithmetic function that takes 3 arguments:
+# the first 2 are numbers, and the third is the operation to be performed on them.
+# If the third argument is +, add them up;
+# if -, then subtract; * - multiply; / - divide (first to second).
+# Otherwise, return the string Not known operation: {operation}.
+# Describe the function in the attached file in such a way that all checks in the __main__ task_1 block are performed correctly.
+# DO NOT CALL THE FUNCTION YOURSELF I HAVE ALREADY DONE THIS IN "assert" STATEMENTS
+
+def arithmetic(left_operand: int, right_operand: int, operation: str) -> int | float | str:
+    if operation == '+':
+        return left_operand + right_operand
+    elif operation == '-':
+        return left_operand - right_operand
+    elif operation == '*':
+        return left_operand * right_operand
+    elif operation == '/':
+        return left_operand / right_operand
+    else:
+        return f'Not known operation: {operation}'
+
+
+if __name__ == "__main__":
+
+    assert arithmetic(3, 4, operation="*") == 12
+    assert arithmetic(3, 4, operation="+") == 7
+    assert arithmetic(25, 5, operation="/") == 5
+    assert type(arithmetic(25, 5, operation="/")) == float
+    assert arithmetic(5, 5, operation="//") == f"Not known operation: //"
+    assert arithmetic.__doc__ == (
+        f"\n{' ' * 8}"
+        f"Apply arithmetic operation for provided left and right operands\n"
+        f"{' ' * 4}"""
+    )
+    assert arithmetic.__code__.co_name == "arithmetic"
+    assert arithmetic.__code__.co_varnames == ("left_operand", "right_operand", "operation")
+    try:
+        arithmetic(1, 2, 3)
+    except TypeError as e:
+        assert e.__class__ is TypeError
+    try:
+        arithmetic(left_operand=1, right_operand=2, operation="+")
+    except TypeError as e:
+        assert e.__class__ is TypeError
+
+    try:
+        arithmetic(1, right_operand=2, operation="+")
+    except TypeError as e:
+        assert e.__class__ is TypeError
```

Comment:
this code should run without AssertionErrors. But I get
![image](https://user-images.githubusercontent.com/25909810/216339560-070d6ed9-9abe-4be4-b949-39372a6c60a8.png)